setup.cfg

```diff
@@ -1,3 +1,78 @@
-# Inside of setup.cfg
 [metadata]
-description-file = README.md
+name = scib
+version = 0.2.0
+author = Malte Luecken, Maren Buettner, Daniel Strobl, Michaela Mueller
+author_email = malte.luecken@helmholtz-muenchen.de, michaela.mueller@helmholtz-muenchen.de
+description = Evaluate single cell integration methods
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = MIT
+url = https://github.com/theislab/scib
+project_urls =
+    Pipeline = https://github.com/theislab/scib-pipeline
+    Reproducibility = https://theislab.github.io/scib-reproducibility
+Bug Tracker = https://github.com/theislab/scib/issues
```

Comment:
do you need an underline here between "Bug" and "Tracker"?

Reply:
It should be fine without. See here for a demo https://test.pypi.org/project/scib/